templates/web/base/admin/bodies/contact-form.html

```diff
@@ -40,7 +40,7 @@
         [% loc('An email address or service ID (Open311 or similar).') %]
       [% END %]
     </p>
-    <input type="text" class="form-control" id="destination" aria-describedby="destination-hint" name="email" size="30" value="[% contact.email | html %]" required>
+    <input type="email" class="form-control" id="destination" aria-describedby="destination-hint" name="email" size="30" value="[% contact.email | html %]" required>
```

Comment:
Ah this one is sometimes not an email, look at the label just above.

Reply:
I added a conditional to tackle this.